incubator/riemann/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
```

Comment:
We've started namespacing templates. Please see #1785 and apply accordingly.

Reply:
Ok, thanks for the feedback. This bit came from `helm create` in version 2.6.2. I just checked 2.7.0 and it produces the same output. If there isn't a PR for that, I'll look into it later.

Reply:
Disregard the commentary on `helm create`. I changed the version linking locally and forgot. helm 2.7.0 outputs properly namespaced template helpers.